management/virtualmachine/resourceextensions_test.go

```diff
@@ -0,0 +1,38 @@
+package virtualmachine
+
+import (
+	"flag"
+	"testing"
+
+	"github.com/MSOpenTech/azure-sdk-for-go/management/testutils"
+)
+
+func init() {
+	flag.BoolVar(&extraVerbose, "resourceextensions.verbose", false, "List all resource extensions")

```

Comment:
This is not good practice for test pkgs. test logs don't get printed unless -test.v exists. Make use of that if u can